cmd/validators/validators.go

```diff
@@ -48,25 +48,14 @@ func CmdError(err error, commandPath string) error {
 	return errors.New(errorMessage)
 }
 
-func getCommandPath(cmd *cobra.Command) string {
```

Comment:
Sneaking this in here based on [this comment](https://github.com/launchdarkly/ldcli/pull/195/files#r1575390041).